ui/index.jsx

```diff
@@ -289,10 +289,13 @@ function AppWrapper() {
               </ErrorBoundary>
             </ConnectedRouter>
           ) : (
+            <Fragment>
             <SplashScreen
               authenticate={() => app.store.dispatch(doAuthenticate(pjson.version))}
               onReadyToLaunch={() => setReadyToLaunch(true)}
             />
+            <SnackBar />
+            </Fragment>
```

Comment:
I don't think the precommit script ran for you, `<SplashScreen` and `<Snackbar />` should be indented.